lib/tandem/tandemSimulator.js

```diff
@@ -79,6 +84,13 @@ exports.make = function(config){
     basal: function(event){
       ensureTimestamp(event);
       if (currBasal != null) {
+        if(currBasal.deliveryType === 'temp') {
+          if (currTempBasal !== null) {
+            currBasal.percent = currTempBasal.percent;
```

Comment:
@jebeck Are you not seeing them anywhere? They're coming through for me:

<img width="1000" alt="screen shot 2015-11-30 at 11 16 10" src="https://cloud.githubusercontent.com/assets/418233/11470427/69ad9e9e-9755-11e5-887f-9449c4a17d59.png">